docs/_ug/Installation.md

```diff
@@ -9,8 +9,12 @@ If you haven't installed FoodRem, simply follow these steps to set it up:
 1. Copy the jar file to an empty folder. This will be your home folder for FoodRem.
 1. Double-click on the jar file to launch FoodRem.
 
-```note 
-If FoodRem does not launch after double-clicking the jar file, check out the [Troubleshooting](#troubleshooting) section for an alternative solution.
+```warning
+Remember to add `foodrem.jar` into an empty folder as additional files will be created when launching FoodRem for the first time! 
```

Comment:
Might be good to explain "additional files", e.g. "additional data and configuration files"